src/Raven.Server/Documents/Sharding/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForBackupDatabaseNow.cs

```diff
@@ -15,29 +21,72 @@ public ShardedOngoingTasksHandlerProcessorForBackupDatabaseNow([NotNull] Sharded
         {
         }
 
-        protected override ValueTask<bool> ScheduleBackupOperationAsync(long taskId, bool isFullBackup, long operationId, DateTime? _)
+        protected override async ValueTask<(long, bool)> ScheduleBackupOperationAsync(long taskId, bool isFullBackup, long operationId, bool __, DateTime? _)
         {
             var token = RequestHandler.CreateTimeLimitedBackgroundOperationToken();
 
+            // backup might be already running on one of the shards, get the old operation id if so
+            var actualOperationId = await RequestHandler.ShardExecutor.ExecuteParallelForAllAsync(new ShardedGetOperationIdForBackupOperation(RequestHandler.HttpContext, taskId, operationId, isFullBackup));
+            
+            // if we got the old operation id then the old backup is still running on one of the shards, we want to keep tracking it
+            bool inProgressOnAnotherShard = actualOperationId != operationId;
+
+            // if only some of the shards are running this backup, 
             var startTime = SystemTime.UtcNow;
-            var t = RequestHandler.DatabaseContext.Operations.AddRemoteOperation<OperationIdResult<StartBackupOperationResult>, ShardedBackupResult, ShardedBackupProgress>(operationId,
+            var t = RequestHandler.DatabaseContext.Operations.AddRemoteOperation<OperationIdResult<StartBackupOperationResult>, ShardedBackupResult, ShardedBackupProgress>(actualOperationId,
                 Server.Documents.Operations.OperationType.DatabaseBackup,
                 "Backup of sharded database : " + RequestHandler.DatabaseName,
                 detailedDescription: null,
-                (_, shardNumber) => new StartBackupOperation.StartBackupCommand(isFullBackup, taskId, operationId, startTime),
+                (_, shardNumber) => new StartBackupOperation.StartBackupCommand(isFullBackup, taskId, actualOperationId, inProgressOnAnotherShard, startTime),
                 token);
-
+            
             t.ContinueWith(_ =>
```

Comment:
can you do `_ = t.ContinueWith()` here to avoid warning?

Reply:
done